dev-docs/modules/userId.md

```diff
@@ -354,6 +354,8 @@ The `request.userId.lipb` object would then look like
 
 Therefore, the adapters can then be implemented to use the `lipibid` as the identifier, and `segments` to which that identifier is associated with.
 
+For identity resolution to work for a specific publisher, LiveIntent has to build a model on the backend. In order to collect data for this model an additional call is issued on each page load.
```

Comment:
I suggest editing the wording here, Something like "LiveIntents identity resolution service initiates an http request on each page load to our identity service, we do this post auction so auctions and the user experience is not impacted. "